scripts/cluster-auth.sh

```diff
@@ -1,49 +1,6 @@
 #!/bin/bash
 set -euo pipefail
 
-# globals
```

Comment:
Is the permission change expected ? Do you need to do a `+x` ?

Reply:
removed the +x permission.

Reply:
removed the `+x`, not needed.